src/d3a/d3a_core/live_events.py

```diff
@@ -55,19 +55,21 @@ def apply(self, area):
         if area_type is not None and area_type != "Area":
```

Comment:
Not part of this PR, but I've seen it in several lines of the codebase so I'll ask. Are we specifically checking for `None` values, or are we checking for broader false values instead? I would usually prefer `if area_type and area_type != 'Area'`, to consider cases in which `area_type` is something like `False`, `""` or `[]`.

Reply:
Here we are actually checking for None. But you are right, there are cases, where these checks should test for   `False`, `""` or `[]`, too. PLease help us to make this consistent. Thanks!